src/main/java/task/Task.java

```diff
@@ -0,0 +1,37 @@
+package task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        if (isDone) {
+            return "\u2713";
+        } else {
+            return "\u2718";
+        }
+    }
+
+    public String getTypeOfTask() {
+        return "NIL";
+    }
+
+    public String getFormattedDescription(){
+        return this.description;
+    }
+
+    public void markAsCompleted() {
```

Comment:
this level of abstraction should stop at marking task as complete, and exclude the print statements. Including the print statements sort of contaminated the level of abstraction. You get what I mean?

Reply:
Yes. I got it thanks